### PR TITLE
Publish system tests for ISIS SANS

### LINKED ISSUE #19364: Publish system tests for ISIS SANS
In this issue we publish a suite of sytem tests which have been ported from the old reduction back-end to the new reduction back-end

This is part of #15875

### PR DESCRIPTION
**Pull request by AntonPiccardoSelg**
Fixes #19364

# ~~IMPORTANT~~

~~Do not merge before #19685 has been merged~~

This PR exposes a new set of system tests for the new ISIS SANS reduction backend.

**To test:**
Please code review. These changes are part of larger set of complete changes which are currently being tested by the ISIS SANS team.

*Does not need to be in the release notes.*
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [x] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/tests/analysis/SANSMergedDetectorsTest_V2.py
Comment (DTasev) [inline Testing/SystemTests/tests/analysis/SANSMergedDetectorsTest_V2.py]: Can this be moved to be part of the import at the start of the file?

**Review by dtasev — CHANGES_REQUESTED** [review]

**Comment by DTasev** [comment]
Looks good, I've just added a minor comment.

File: Testing/SystemTests/tests/analysis/SANSMergedDetectorsTest_V2.py
Comment (AntonPiccardoSelg) [inline Testing/SystemTests/tests/analysis/SANSMergedDetectorsTest_V2.py]: Actually, this can be removed :+1:

**Comment by AntonPiccardoSelg** [comment]
There is an issue with `SpectrumNumberTranslator` which I think has changed recently. I could use @SimonHeybrock for this.

**Review by dtasev — APPROVED** [review]

**Comment by DTasev** [comment]
If it's going to be a significant change maybe you could do it in a separate PR? Or I can wait until it's ready before shipping it, your call

**Comment by AntonPiccardoSelg** [comment]
@Dtasev It is not clear what the change will have to be. It might uncover a logical flaw in the core. The fix would be in a separate PR. At this point we cannot ship this PR yet, I will let you know if there is any news on this. Worst case we have to bump this back to 3.11.

**Comment by SimonHeybrock** [comment]
@AntonPiccardoSelg Could you clarify, I am not sure which issue you are referring to.

**Comment by AntonPiccardoSelg** [comment]
@SimonHeybrock I think I have figured it out. Some updates to the `SpectrumNumberTranslator` and surrounding code, have pointed out an inconsistency in the SANS code.

**Comment by AntonPiccardoSelg** [comment]
@DTasev Your change request has been applied, the underlying bug has been resolved in #19686

**Comment by AndreiSavici** [comment]
Is this ready to be merged?

**Comment by DTasev** [comment]
Yes, the changes look good, the related bug has been resolved in another issue. Looks good, happy to :shipit:

**Pull request merged**
